gprofiler/utils.py

```diff
@@ -74,8 +75,8 @@ def get_process_nspid(pid: int) -> Optional[int]:
     # from the listing of those files itself.
     return None
 
-
-def start_process(cmd: Union[str, List[str]], via_staticx: bool, **kwargs) -> Popen:
+libc = ctypes.CDLL('libc.so.6')
```

Comment:
Should not be initialized upon import (as happens here). See how it's done in `ns.py` of granulate-utils.

Reply:
done